draft-ietf-quic-http.md

```diff
@@ -2026,6 +2026,8 @@ The "h3" string identifies HTTP/3:
 New registries created in this document operate under the QUIC registration
 policy documented in Section 22.1 of {{QUIC-TRANSPORT}}.  These registries all
 include the common set of fields listed in Section 22.1.1 of {{QUIC-TRANSPORT}}.
+These registries \[SHALL be/are] collected under a "Hypertext Transfer Protocol
```

Comment:
Is `\[SHALL be/are]` form language?

Reply:
Basically invites the RFC editor to change it to "are" in the final document, but for the time being, it's a direction to IANA rather than a statement of fact.  Transport does the same, IIRC.